frontend/packages/console-shared/src/utils/icon-utils.ts

```diff
@@ -0,0 +1,7 @@
+import * as _ from 'lodash';
+import * as operatorLogo from '../images/operator.svg';
+
+export const getImageForCSVIcon = (csv) => {
+  const icon = _.get(csv, 'spec.icon', []);
```

Comment:
FWIW, I think you can do this now:
```suggestion
  const icon = csv?.spec?.icon || [];
```

Reply:
Thanks for the suggestion. I was not aware that typescript has been upgraded in console.

Reply:
Well, we also need to update prettier to be able to use this feature.

Reply:
Ah interesting, I turned off my watcher for my review version of OpenShift... does it to something crazy?

Reply:
Its throwing an error saying `expression expected` after the `?.`

Reply:
Hmmm ... I didn't see any inline eslint errors when I wrote it :/ ah well, maybe for a future time then.

Reply:
Now, i've broken `eslint-prettier-plugin` trying to update `prettier`. Should look at this in another PR. I can raise one after this.

Reply:
Don't bother with it. There's already a PR up to upgrade eslint. We cannot use the feature yet.